Parsers/ASimProcessEvent/Parsers/ASimProcessTerminateMicrosoftSysmonWindowsEvent.yaml

```diff
@@ -0,0 +1,59 @@
+Parser:
+  Title: Process Terminate Event ASIM parser for Microsoft Windows Security Events
+  Version: '0.4.1'
+  LastUpdated: June 3, 2024
+Product:
+  Name: Microsoft Windows Events Sysmon
+Normalization:
+  Schema: ProcessEvent
+  Version: '0.1.0'
```

Comment:
ASIM schema "ProcessEvent" version "0.1.0" is incorrect. The correct version for ASIM schema "ProcessEvent" is "0.1.4"

Reply:
done